vcx/docs/getting-started/getting-started.md

```diff
@@ -0,0 +1,646 @@
+﻿# Getting Started with Libindy
```

Comment:
I believe we should change the title to reference VCX

Reply:
Yes :)